bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.2.1
+version: 12.3.0
```

This seems like a bug fix, right? Should we just bump the patch version?

```suggestion
version: 12.2.2
```

---

I wasn't sure, it's not really a bug, more of an enhancement but not quite a feature.